sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/SessionManager.java

```diff
@@ -210,8 +218,9 @@ public synchronized void stop() {
       try {
         backgroundOperationPool.awaitTermination(timeout, TimeUnit.SECONDS);
       } catch (InterruptedException e) {
-        LOG.warn("HIVE_SERVER2_ASYNC_EXEC_SHUTDOWN_TIMEOUT = " + timeout +
-            " seconds has been exceeded. RUNNING background operations will be shut down", e);
+        LOG.warn("HIVE_SERVER2_ASYNC_EXEC_SHUTDOWN_TIMEOUT = {} ms has been exceeded. " +
+          "RUNNING background operations will be shut down", e,
+          MDC.of(LogKeys.TIMEOUT$.MODULE$, timeout * 1000));
```

Comment:
nit: 1000L

Reply:
Done

Reply:
This is not updated..

Reply:
oh timeout is long type already. Keep the current code is also fine